packages/babel-plugin-object-enumerables/README.md

```diff
@@ -0,0 +1,100 @@
+# babel-plugin-object-enumerables
```

Comment:
I'm not 100% on the naming conventions here -- maybe this should be called `babel-plugin-transform-object-enumerables` instead?